router/router.go

```diff
@@ -509,7 +514,14 @@ func (worker *workerT) handleWorkerDestinationJobs() {
 
 			//Not saving payload to DB if transformAt is not "router"
 			if destinationJobMetadata.TransformAt != "router" {
-				payload = []byte(`Same as payload in router table. Use job_id for look up.`)
+				payload = []byte{}
+			}
+
+			var errorResponse []byte
```

Comment:
Move into postStatusOnResponseQ(), save payload for both batched and dest transformed jobs

Reply:
done